librz/core/libs.c

```diff
@@ -51,51 +51,33 @@ CB(parse, parser)
 CB(bin, bin)
 CB(egg, egg)
 
-static void __openPluginsAt(RzCore *core, const char *arg, const char *user_path) {
-	if (arg && *arg) {
-		if (user_path) {
-			if (rz_str_endswith(user_path, arg)) {
-				return;
-			}
-		}
-		char *pdir = rz_str_rz_prefix(arg);
-		if (pdir) {
-			rz_lib_opendir(core->lib, pdir);
-			free(pdir);
-		}
-	}
-}
-
-static void __loadSystemPlugins(RzCore *core, int where, const char *path) {
+static void loadSystemPlugins(RzCore *core, int where) {
 #if RZ_LOADLIBS
-	if (!where) {
-		where = -1;
-	}
-	if (path) {
-		rz_lib_opendir(core->lib, path);
-	}
 	const char *dir_plugins = rz_config_get(core->config, "dir.plugins");
 	if (where & RZ_CORE_LOADLIBS_CONFIG) {
-		rz_lib_opendir(core->lib, dir_plugins);
+		rz_lib_opendir(core->lib, dir_plugins, false);
 	}
 	if (where & RZ_CORE_LOADLIBS_ENV) {
 		char *p = rz_sys_getenv(RZ_LIB_ENV);
 		if (p && *p) {
-			rz_lib_opendir(core->lib, p);
+			rz_lib_opendir(core->lib, p, false);
 		}
 		free(p);
 	}
 	if (where & RZ_CORE_LOADLIBS_HOME) {
-		char *hpd = rz_str_home(RZ_HOME_PLUGINS);
-		if (hpd) {
-			rz_lib_opendir(core->lib, hpd);
-			free(hpd);
-		}
+		char *hpd = rz_path_home(RZ_PLUGINS);
+		rz_lib_opendir(core->lib, hpd, false);
+		free(hpd);
+
+		// TODO: remove after 0.4.0 is released
```

Comment:
Don't forget to open an issue to track this after, please

Reply:
https://github.com/rizinorg/rizin/issues/2046